app/src/main/res/layout/customise_card_dialog.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/customiseCardDialog"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:padding="@dimen/card_padding"
+        android:theme="@style/AppTheme.AlertDialogTheme">
+
+        <TextView
+            android:id="@+id/customiseCardTitleTextView"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/dialog_header_top_margin"
+            android:text="Customise card"
+            android:textAppearance="@style/TextAppearance.Headline6"
+            app:layout_constraintBottom_toTopOf="@id/customiseCardScrollView"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:layout_editor_absoluteX="10dp" />
+
+        <ScrollView
+            android:id="@+id/customiseCardScrollView"
+            android:layout_width="398dp"
+            android:layout_height="160dp"
```

Comment:
Ideally don't hardcode any dimensions in the dialog if you can avoid it. Just use wrap_content/match_parent